test/test_eos.py

```diff
@@ -302,7 +300,7 @@ def inf_norm(x):
         pyro_t = pyro_mechanism.get_temperature(pyro_e, tin, yin)
         pyro_p = pyro_mechanism.get_pressure(pyro_m, pyro_t, yin)
         pyro_h = pyro_mechanism.get_mixture_enthalpy_mass(pyro_t, yin)
-        pyro_s = pyro_mechanism.get_mixture_entropy_mass(pyro_p, pyro_t, yin)
+        # pyro_s = pyro_mechanism.get_mixture_entropy_mass(pyro_p, pyro_t, yin)
```

Comment:
OOC, how's come?

Reply:
Per Pyrometheus review, we removed entropy from its native functions. I am wondering if I should remove it completely or add local functions somewhere to keep the verification.

Reply:
If it is no longer in the base package and the corresponding test was removed there, then this is the right thing to do. I was just curious about the removal.

Reply:
Removed. :+1: